src/NuGet.Core/NuGet.Commands/ClientCertificatesCommand/ModifyClientCertArgs.cs

```diff
@@ -0,0 +1,168 @@
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Security.Cryptography.X509Certificates;
+using NuGet.Common;
+
+namespace NuGet.Commands
+{
+    public abstract class ModifyClientCertArgs : BaseClientCertArgs
+    {
+        /// <summary>
+        ///     FindBy added to a storage client certificate source
+        /// </summary>
+        public string FindBy { get; set; }
+
+        /// <summary>
+        ///     FindValue added to a storage client certificate source
+        /// </summary>
+        public string FindValue { get; set; }
+
+        /// <summary>
+        ///     Indicates that action forced
+        /// </summary>
+        public bool Force { get; set; }
+
+        /// <summary>
+        ///     Name of the package source.
+        /// </summary>
+        public string PackageSource { get; set; }
+
+        /// <summary>
+        ///     Password for the certificate, if needed. This option can be used to specify the password for the certificate.
+        /// </summary>
+        public string Password { get; set; }
+
+        /// <summary>
+        ///     Path to certificate file added to a file client certificate source
+        /// </summary>
+        public string Path { get; set; }
+
+        /// <summary>
+        ///     StoreLocation added to a storage client certificate source
+        /// </summary>
+        public string StoreLocation { get; set; }
+
+        /// <summary>
+        ///     StoreName added to a storage client certificate source
+        /// </summary>
+        public string StoreName { get; set; }
+
+        /// <summary>
+        ///     Enables storing password for the certificate by disabling password encryption.
+        /// </summary>
+        public bool StorePasswordInClearText { get; set; }
+
+        public override void Validate()
+        {
+            if (!IsPackageSourceSettingProvided())
+            {
+                throw new CommandLineArgumentCombinationException(string.Format(CultureInfo.CurrentCulture,
+                                                                                Strings.Error_PropertyCannotBeNullOrEmpty,
+                                                                                nameof(PackageSource)));
+            }
+
+            if (IsFileCertSettingsProvided() == IsStoreCertSettingsProvided())
+            {
+                throw new CommandLineArgumentCombinationException(string.Format(CultureInfo.CurrentCulture,
+                                                                                Strings.Error_CouldNotUpdateClientCertificate,
+                                                                                Strings.Error_InvalidCombinationOfArguments));
+            }
+        }
+
+        public X509FindType? GetFindBy()
+        {
+            if (Enum.TryParse("FindBy" + FindBy, true, out X509FindType value))
```

Comment:
Add argument names to values that are not obvious:

```suggestion
            if (Enum.TryParse("FindBy" + FindBy, igoreCase: true, out X509FindType value))
```